File: UnihackFront/src/app/entities/prescription.ts
import { Case } from './case';

export class Prescription {
    id: string;                    // Primary key as string
    prescriptionName: string;      // Prescription name
    prescriptionDescription: string; // Prescription description
    prescriptionResult: string;    // Prescription result

    parentCase: Case;              // Parent Case to which the prescription belongs
}

File: UnihackBack/src/main/java/com/unihackback/entity/Prescription.java
package com.unihackback.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.ManyToOne;

@Entity
public class Prescription {

    @Id
    private String id;

    private String prescriptionName;
    private String prescriptionDescription;

    private String prescriptionResult;

    @ManyToOne
    private Case parentCase;
}
